spec/tapioca/compilers/dsl/active_record_columns_spec.rb

```diff
@@ -292,31 +286,29 @@ def money_column=(value); end
         end
 
         it("falls back to generating BigDecimal for money column if MoneyColumn is not defined") do
-          files = {
-            "file.rb" => <<~RUBY,
-              require "rails/railtie"
-              require "money"
-
-              class Post < ActiveRecord::Base
-                money_column(:money_column, currency: "USD")
+          add_ruby_file("schema.rb", <<~RUBY)
+            ActiveRecord::Migration.suppress_messages do
+              ActiveRecord::Schema.define do
+                create_table :posts do |t|
+                  t.decimal :money_column
+                end
               end
+            end
+          RUBY
 
-              # Make `MoneyColumn` disappear artifically
-              Object.send(:remove_const, :MoneyColumn)
-            RUBY
+          add_ruby_file("post.rb", <<~RUBY)
+            require "rails/railtie"
+            require "money"
 
-            "schema.rb" => <<~RUBY,
-              ActiveRecord::Migration.suppress_messages do
-                ActiveRecord::Schema.define do
-                  create_table :posts do |t|
-                    t.decimal :money_column
-                  end
-                end
-              end
-            RUBY
-          }
+            class Post < ActiveRecord::Base
+              money_column(:money_column, currency: "USD")
+            end
+
+            # Make `MoneyColumn` disappear artifically
+            Object.send(:remove_const, :MoneyColumn)
+          RUBY
 
-          output = rbi_for(:Post, files)
+          output = rbi_for(:Post)
 
           expected = indented(<<~RUBY, 4)
```

Comment:
```suggestion
          expected = indented(<<~RBI, 4)
```
For consistency

Reply:
Good catch! I found one more instance of it.